### DISCUSSION TITLE
[Feature] custom logout request

### DISCUSSION THREAD

**Discussion by ModProg** [immich-app/immich#12632] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have setup oauth with Authelia so logging out of immich does little to nothing without also logging out of Authelia.

Therefor, I'd like to have `Logout` trigger a custom endpoint:

`https://authelia.example.de/api/logout`

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by olivluca** [comment]
I think that's an issue with authelia, since it doesn't provide an end_session_endpoint

https://github.com/immich-app/immich/blob/a11d45458b3fef322afbdb049a95658ec7aebd96/server/src/repositories/oauth.repository.ts#L29

**Reply by ModProg** [reply]
Oh so this is an oauth feature? I'll ask in authelia concerning this.

**Comment by olivluca** [comment]
Well, I'm not sure, I just checked the reply from .well-known/openid-configuration and I saw that authelia doesn't provide the end_session_endpoint while, e.g., lemonldap does (but I'm not using lemonldap with immich).
In any case I found this:
https://github.com/authelia/authelia/issues/5057

**Comment by bo0tzz** [comment]
#11104
